search/search/harvester.py

```diff
@@ -11,13 +11,41 @@
 from search.logger import logger
 
 
+def convert_bucket_name_if_s3prefix(bucket_name: str) -> str:
+    if settings.s3_prefix:
+        return f"{settings.s3_prefix}-{bucket_name}"
+    else:
+        return bucket_name
+
+
+class NotConfiguredException(Exception):
+    pass
+
+
+def create_boto3_config():
+    boto3_config = {}
+    if settings.s3_credentials_provider == "minio":
+        logger.info(f"S3_Credentials provider - {settings.s3_credentials_provider}")
+        boto3_config.update(
+            {
+                "aws_access_key_id": settings.s3_login,
+                "aws_secret_access_key": settings.s3_pass,
+                "endpoint_url": settings.s3_endpoint_url,
+            }
+        )
+    elif settings.s3_credentials_provider == "aws_iam":
+        logger.info(f"S3_Credentials provider - {settings.s3_credentials_provider}")
```

Comment:
Let's move logging to a single line (L43)

Reply:
agreed